src/sidebar/util/test/observe-element-size-test.js

```diff
@@ -2,9 +2,28 @@
 
 const observeElementSize = require('../observe-element-size');
 
+/**
+ * Wait for a condition to become true.
+ *
+ * @param {() => boolean} callback
+ */
+function waitFor(callback) {
+  return new Promise(resolve => {
+    const timer = setInterval(() => {
+      if (callback()) {
+        clearInterval(timer);
+        resolve();
+      }
+    }, 0);
+  });
+}
```

Comment:
A conceptually similar utility was recently added to the lms repo. It seems likely we may want to package up some of our frequently used test utilities into a shared package.